waku/v2/node/storage/message/sqlite_store/queries.nim

```diff
@@ -247,11 +247,11 @@ proc timeRangeWhereClause(startTime: Option[Timestamp], endTime: Option[Timestam
 
   var where = "("
   if startTime.isSome():
-    where &= "senderTimestamp >= (?)"
+    where &= "receiverTimestamp >= (?)"
   if startTime.isSome() and endTime.isSome():
     where &= " AND "
   if endTime.isSome():
-    where &= "senderTimestamp <= (?)"
+    where &= "receiverTimestamp <= (?)"
   where &= ")"
   some(where)
```

Comment:
The SQLite store implementation now will query by `receiverTimestamp` field. 

This field will be renamed `storeTimestamp`  or `storedAt` in one of the subsequent PRs to avoid any future confusion.